paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -252,7 +255,7 @@ internal class CustomerSheetViewModel @Inject constructor(
                             }
 
                             removedPaymentSelection
-                        },
+                        } ?: originalPaymentSelection,
```

Comment:
What happens if you remove the original selection?

Reply:
Then nothing should be selected.

Reply:
Oh, `originalPaymentSelection` is being set to `null`. I find it a little odd that we’re changing an original value. Semantically, an original value can’t change. Should this be called `currentPaymentSelection` then? Though it seems like we already have that as another property…

Reply:
Good point, I will rethink that!

Reply:
I am removing currentSelection in the next PR. So, I can rename original to something more appropriate in a future PR

Reply:
Renamed here: https://github.com/stripe/stripe-android/pull/7078